vcpkg.json

```diff
@@ -16,7 +16,14 @@
     "directxmath",
     "directxtk",
     "rapidcsv",
-    "spdlog"
+    "spdlog",
+    "openvr"
+  ],
+  "overrides": [
+    {
+      "name": "fmt",
+      "version": "10.2.1"
```

Comment:
I've tried not to pin fmt. I thought clib actually stripped fmt out as we use std::format. Was that a wrong assumption? I'm on the fence about pinning it to a version though. I get that fmt breaks shit every version release, but I do try to update dependencies unless there's a strong reason not to. Thoughts?

Reply:
Since we're using `spdlog` through vcpkg there's no way to avoid the `fmt` dependency. The latest version of `fmt` (v11) would require some migration work--I'd be happy to look into it in a future PR